src/app-sandbox/container.test.tsx

```diff
@@ -1,3 +1,7 @@
+/**
+ * @jest-environment jsdom
```

Comment:
Is this part necessary? jsdom is a lot slower, I believe.

Reply:
The test suite fails if I don't define it. 

Test output:
```
 FAIL  src/app-sandbox/container.test.tsx
  ● Test suite failed to run

    The error below may be caused by using the wrong test environment, see https://jestjs.io/docs/en/configuration#testenvironment-string.
    Consider using the "jsdom" test environment.
    
    ReferenceError: document is not defined

      43 | export class Container extends React.Component<Properties, State> {
      44 |   static defaultProps = {
    > 45 |     document,
         |     ^
      46 |   };
      47 |
      48 |   static mapState(state: RootState): Partial<Properties> {
```

Reply:
Maybe we just shouldn't have a default?